src/main/scala/vsys/contract/ContractPermitted.scala

```diff
@@ -205,8 +207,8 @@ object ContractPermitted {
     val opcTDBARBalanceOfIndex: Array[Byte] = Array(DataStack.balanceOfInput.addressIndex, DataStack.balanceOfInput.tokenIndex)
 
     // init
-    val initOpc: List[Array[Byte]] = List(OpcId.opcLoadSigner, OpcId.opcCDBVSet, OpcId.opcCDBVSet, OpcId.opcTDBNewToken)
-    val initOpcIndex: List[Array[Byte]] = List(opcLoadSignerIndex, opcCDBVSetIssuerInitIndex, opcCDBVSetMakerIndex, opcTDBNewTokenIndex)
+    val initOpc: List[Array[Byte]] = List(OpcId.opcLoadSigner, OpcId.opcCDBVSet, OpcId.opcCDBVSet, OpcId.opcCDBVSet, OpcId.opcTDBNewToken)
+    val initOpcIndex: List[Array[Byte]] = List(opcLoadSignerIndex, opcCDBVSetIssuerInitIndex, opcCDBVSetMakerIndex, opcCDBVSetDescIndex, opcTDBNewTokenIndex)
```

Comment:
`OpcId.opcCDBVSet` to set the contract description is useless here.

since contract description is register contract transaction's attachment. we can get the description by using transaction id directly. 

p.s. 
1. `contract id -> transaction id`
2. `transaction id -> full transaction`
3. `transaction's attachment`

Reply:
But we decide to use data entry for all descriptions including contract description